cypress/e2e/RoundTrips/SimpleProject.feature

```diff
@@ -0,0 +1,473 @@
+  ## 800 Filter by text and verify that some diagrams disappear
+  ## 801 Clear filter text and verify all diagrams are present
+  ## 802 Select Infrastructure tag and verify only all Infrastructure diagrams are present
+  ## 803 Select CI tag and verify all diagrams are present
+  ## 804 Unselect Infrastructure tag and verify only all CI diagrams are present
+  ## 805 Unselect CI tag and verify all diagrams are present
+
+  ################## AddComponent.feature ##################
+  ## 900 Plugin test installed in component definitions list (Draw view) should not create project configuration file (Text view)
+  ## 901 Add a component (Draw view) should create project configuration file (Text view)
+  ## 902 Update plugin file content with a new object (Text view) should display the corresponding plugin component (Draw view)
+  ## 903 Link two components (Draw view) should update plugin file content with new attributes properties (Text view)
+  ## 904 Add link attributes inside plugin file content (Text view) should display link between two components (Draw view)
+
+  Scenario: Roundtrip about simple project
+
+    Given I clear cache
+    And   I set viewport size to '1920' px for width and '1080' px for height
+    And   I set context field 'projectName' with 'localProjectTest'
+    And   I set context field 'remoteProjectName' with 'leto-modelizer-project-test'
+    And   I set context field 'remoteProjectUrl' with 'https://github.com/ditrit/leto-modelizer-project-test'
+    And   I set context field 'firstModelName' with 'firstModelTest'
+    And   I set context field 'firstModelFolder' with 'model1'
+    And   I set context field 'firstModelFile' with 'model1/main.tf'
+    And   I set context field 'secondModelFolder' with 'model2'
+    And   I set context field 'secondModelName' with 'secondModelTest'
+    And   I set context field 'secondModelFile' with 'model2/main.tf'
+    And   I set context field 'thirdModelName' with 'thirdModelTest3Githubator.yml'
+    And   I set context field 'modelRenamed' with 'newModelTest'
+    And   I visit the '/'
+
+    ## 100 Home page should display an empty project message and one template project card
+    Then I expect '[data-cy="project-grid-empty"]' exists
+    And  I expect '[data-cy="project-grid-empty"]' is 'No projects, please create a new project to have one here 😉'
+    And  I expect '[data-cy="template-card_project_template"]' exists
+    And  I expect '[data-cy="template-card_project_template"] [data-cy="title-container"]' is 'Project template'
+
+    ## 101 Creating a project with an empty name should display an error
+    When I click on '[data-cy="create-project-button"]'
+    And  I click on '[data-cy="create-project-form"] [data-cy="submit-button"]'
+    Then I expect '[data-cy="create-project-form"] [role="alert"]' is 'Please type something'
+
+    And I visit the '/'
+    And I expect '[data-cy="create-project-button"]' exists
```

Comment:
Is it really necessary ? (lines 84-85)

Reply:
To me its better than adding a wait instead. But I will try to replace all "visit the /" by the click on the logo, it should be better.

Reply:
FYI, I replaced all the "I visit the /" by a click on the logo whenever I could